src/main/java/com/commercetools/sync/inventories/SkuKeyTuple.java

```diff
@@ -0,0 +1,116 @@
+package com.commercetools.sync.inventories;
+
+import io.sphere.sdk.inventory.InventoryEntry;
+import io.sphere.sdk.inventory.InventoryEntryDraft;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+import static org.apache.commons.lang3.StringUtils.isEmpty;
+
+/**
+ * Helper class that holds sku and supply channel key - values used for {@link io.sphere.sdk.inventory.InventoryEntry}
+ * distinction. Presence of sku and key in created instance is similar as for inventory entry resource: supply channel
+ * key can be {@code null} value since it is optional in inventory entry, but sku has to be not null neither not empty.
+ * Can be used as key in Maps.
+ */
+final class SkuKeyTuple {
+
+    static final String SKU_NOT_SET_MESSAGE = "Sku is not set";
```

Comment:
actually it is not possible to have an InventoryEntry with an empty sku. It will not throw an error by the platform.